src/types/Operation.ts

```diff
@@ -0,0 +1,16 @@
+export interface Operation {
```

Comment:
As this type is Tezos specific, please move the file to the Tezos package. IIRC, there is already a file containing Tezos types.

Reply:
Roger!